internal/init/init.go

```diff
@@ -85,25 +86,26 @@ func (c *Client) NewCommand() *cobra.Command {
 			fmt.Println()
 			token := string(b)
 			c.Servicer.SetToken(token)
-			metalClient := c.Servicer.API(cmd)
-			c.UserService = metalClient.Users
-			c.ProjectService = metalClient.Projects
+			metalGoClient := c.Servicer.MetalAPI(cmd)
+			c.UserService = *metalGoClient.UsersApi
+			c.ProjectService = *metalGoClient.ProjectsApi
 
-			user, _, err := c.UserService.Current()
+			user, _, err := c.UserService.FindCurrentUser(context.Background()).Execute()
 			if err != nil {
 				return err
 			}
-			organization := user.DefaultOrganizationID
+			organization := user.AdditionalProperties["default_organization_id"]
 			project := ""
```

Comment:
We should initialize `project` with `fmt.Sprintf("%v", user.AdditionalProperties["default_project_id"])` so that we use the default if they have one; we should only run `getFirstProjectId` when the user _doesn't_ already have a default.